### ISSUE TITLE
Japanese localization used for en-GB on macOS 10.15.7

### ISSUE THREAD

**Issue by dnadlinger** [Nheko-Reborn/nheko#470]
As per the title: For whatever reason, the client, from the official binaries, uses the Japanese UI on my macOS 10.15 box, which is set to English/UK as far as the system configuration goes, and

```
LANG="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_CTYPE="UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_ALL=
```

### System

- Nheko version: 0.8.1-1a2517e8
- Installation method: brew
- Operating System: macOS 10.15.7

### Logs

Nothing of interest:

```
$ less ~/Library/Caches/nheko/nheko/nheko.log
[2021-02-12 10:24:30.258] [qml] [warning] Populating font family aliases took 751 ms. Replace uses of missing font family "Default" with one that exists to avoid this cost.  (:0, )
[2021-02-12 10:24:34.944] [ui] [info] jdenticon plugin not found.
[2021-02-12 10:24:34.966] [ui] [info] timelineWidth: -204
[2021-02-12 10:24:34.966] [ui] [info] timelineWidth: 18446744073709551412, min 560
[2021-02-12 10:24:34.968] [ui] [info] timelineWidth: -204
[2021-02-12 10:24:34.968] [ui] [info] timelineWidth: 18446744073709551412, min 560
[2021-02-12 10:24:34.982] [ui] [info] starting nheko 0.8.0-a9faecea
```

**Comment by LorenDB** [comment]
@dnadlinger: can you check to see if this is still an issue with the latest 0.11.3 binary?

**Comment by dnadlinger** [comment]
Seems to be resolved (using the official `nheko-v0.11.3-apple-silicon.dmg` binary build).

**Comment by LorenDB** [comment]
Wonderful!
